orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/model/support/TriggerDeserializer.kt

```diff
@@ -209,6 +219,6 @@ class TriggerDeserializer :
     hasNonNull("pluginId")
 
   private fun JsonNode.looksLikeCustom() =
-    customTriggerSuppliers.any { it.predicate.invoke(this) }
+    customTriggerSuppliers.any { it.predicateByNode.invoke(this) }
```

Comment:
Why did we change it? We just are changing the name? Yea, let's not do that for git history reasons.

Reply:
I changed it because I defined two predicates.

- predicateByNode will receive a JsonNode and will return a boolean if the customDeserializer needs to handle the trigger.
- predicateByTrigger will receive a Trigger and will return a boolean if the customDeserializer needs to handle the trigger.

I use the definition to workaround the issue with the other object in the Trigger.